### ISSUE TITLE
Remove accname/name/comp_text_node.html tests that assume &nbsp trimming

### ISSUE THREAD

**Issue by dandclark** [web-platform-tests/interop#649]
### Test List

The following tests in [accname/name/comp_text_node.html](https://wpt.fyi/results/accname/name/comp_text_node.html?label=experimental&label=master&aligned&view=interop&q=label%3Ainterop-2024-accessibility):
- span[role=button] with text node, with leading/trailing non-breaking space
- div[role=heading] with text node, with leading/trailing non-breaking space
- button with text node, with leading/trailing non-breaking space
- heading with text node, with leading/trailing non-breaking space
- link with text node, with leading/trailing non-breaking space

### Rationale

The tests all validate that `&nbsp;` at the beginning and end of a node's computed name is trimmed, e.g.:

`<span role="button" tabindex="0" class="ex" data-expectedlabel="button&nbsp;label" data-testname="span[role=button] with text node, with leading/trailing non-breaking space">&nbsp;button&nbsp;label&nbsp;</span>`
Chromium and Safari fail the tests because they do not trim the `&nbsp;`s.

The [accname computation steps](https://w3c.github.io/accname/#computation-steps) do not say anything about trimming whitespace from the computed result. I found an older issue seeking clarification about this that is not yet resolved: https://github.com/w3c/accname/issues/95. The WPT utils used for these tests do some of their own whitespace normalizing (added [here](https://github.com/web-platform-tests/wpt/pull/42407/files#diff-6870d82f11ff11cf7c7b544756cecfdac2046acbfe2fbb0640e6d415fbf99916)) but that explicitly does not include non-breaking spaces.

Given that the behavior validated by the tests is not stated in the spec, and #95 does not seem close to a resolution, I'm proposing that these tests be removed from Interop2024.

cc @cookiecrook

**Comment by jgraham** [comment]
@zcorpan @jcsteh do you have any input on this one?

**Comment by cookiecrook** [comment]
Who can move this to https://github.com/web-platform-tests/interop-accessibility ? IIRC the test predates a spec clarification that it should be ASCII whitespace.
